src/Components/Toast.js

```diff
@@ -0,0 +1,14 @@
+import { toast, Slide } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+export default function ({ message, type }) {
+  console.log("Comes Here");
+  return toast(message, {
+    position: toast.POSITION.TOP_CENTER,
+    autoClose: 1000,
+    type: type,
+    transition: Slide,
+    closeOnClick: true,
+    pauseOnHover: false,
```

Comment:
Toast can be directly invoked by an inbuilt function called **toast**. Check this [LINK](https://www.knowledgehut.com/blog/web-development/react-toastify)

Reply:
Thank you for sharing it :) I will work with it in next project